setup.py

```diff
@@ -34,8 +34,9 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
+        'django-private-storage==2.2.2',
```

Comment:
use a dependency range, eg `~=2.2`(will install latest version in 2.2.x), I'd move it after openwisp-controller, is there a reason you put it first?

Reply:
Yeah, to keep it alphabetically sorted. Is there any preference?

Reply:
Main openwisp dependencies first and then the rest